src/main/java/com/nylas/Event.java

```diff
@@ -23,6 +23,7 @@
 	private String status;
 	private Boolean read_only;
 	private Boolean busy;
+	private Map<String, Object> metadata;
```

Comment:
Since metadata appears to be limited to string values, I think it's best to model it is as a `Map<String, String>` instead,

https://developer.nylas.com/docs/connectivity/calendar/event-metadata/#event-metadata-object

Reply:
Good call! Thanks for the link!